spec/models/user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :encrypted_password }
+  end
```

Comment:
You need to test that the FactoryBot is valid here

Reply:
Factory bot configured and unit tests added and going green